myapp/app/views/users/index.html.erb

```diff
@@ -20,8 +22,10 @@
           <th class="text-center"><%= user.id %></th>
           <th class="w-25"><%= user.name %></th>
           <th class="w-25"><%= user.email %></th>
+          <th class="text-center"><%= user.is_admin? %></th>
           <th class="text-center"><%= user.created_at.strftime("%Y/%m/%d %H:%M")  %></th>
           <th class="text-center"><%= user.updated_at.strftime("%Y/%m/%d %H:%M")  %></th>
+          <th class="text-center"><%= user.tasks.length %></th>
           <th class="text-center"><%= link_to I18n.t('action.show'), user_path(user), class:"btn btn-primary" %></th>
           <th class="text-center"><%= link_to I18n.t('action.edit'), edit_user_path(user), class:"btn btn-primary" %></th>
           <% if user.id != @current_user.id %>
```

Comment:
- I changed to display the number of tasks of each users.
<img width="1502" alt="image" src="https://user-images.githubusercontent.com/37566073/225510875-30e126ae-b63d-411a-b00f-9dffdf6b375d.png">